src/main/java/com/carrotcreative/cream/cache/CacheManager.java

```diff
@@ -40,6 +43,22 @@ private CacheManager(Context context)
         mRootDir = context.getCacheDir();
     }
 
+    public void setLruCacheSize(int lruCacheSize)
```

Comment:
Well it kindof is a boolean thing isn't it?  We're setting `mLruCacheEnabled` to true here.

Yeah I guess the thing that I'm feeling odd about this name is that from a user perspective it doesn't fluently feel like I've ever enabled the cache.

```java
// At this point LRU Cache is disabled
`CacheManager.getInstance().setLruCacheSize(size)`
// Now LRU cache is enabled
```

As a user I might feel like the cache was enabled before (with a default cache size) and I've just set the size to a custom size.

Perhaps this isn't the method to do it, but I really do want a boolean-sounding method that does this line here: 

```
mLruCacheEnabled = true;
```